src/control/nav2_dwa_controller/test/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+set(TEST_LAUNCH_DIR ${CMAKE_CURRENT_SOURCE_DIR}/test_launch_files)
```

Comment:
If there are tests that are common to controller (FollowPath) implementations they could be at navigation2/src/nav2_control_tests (a peer to the implementations).

Reply:
Removed this change